src/components/inputs/money-input/money-input.js

```diff
@@ -31,6 +31,11 @@ const createCurrencySelectStyles = ({
     }),
     singleValue: base => ({
       ...base,
+      marginLeft: '0',
+      marginRight: '0',
+      maxWidth: '100%',
+      left: '50%',
+      transform: 'translate(-50%, -50%)',
```

Comment:
In `control` we set `width: '72px',`. This is what I did originally to get the `react-select` to be visible at all and to avoid the currency ellipses. Can we remove that now? Or would it maybe have been enough to just bump that to `75px`?